source/docs/h2_metadata.md

```diff
@@ -48,7 +48,7 @@ create a new filter.
 If Envoy needs to parse a metadata sent on a request from downstream to upstream, a
 StreamDecodeFilter should be created. The interface to override is
 
-FilterMetadataStatus StreamDecoderFilter::decodeMetadata(MetadataMapPtr&& metadata);
+FilterMetadataStatus StreamDecoderFilter::decodeMetadata(MetadataMapPtr& metadata);
```

Comment:
Actually it looks like it's `MetadataMap&` (no Ptr).

https://github.com/envoyproxy/envoy/blob/48c4fb87e9632baa6145a21771fb4d67d4aa4231/include/envoy/http/filter.h#L683

Reply:
updated